ingestion/src/metadata/ingestion/source/metadata/atlas.py

```diff
@@ -200,26 +249,103 @@ def _parse_table_columns(self, table_response, tbl_entity, name) -> List[Column]
 
     def get_database_entity(self, database_name: str) -> Database:
         return Database(
+            id=uuid.uuid4(),
             name=database_name,
-            service=EntityReference(id=self.service.id, type=self.config.service_type),
+            service=EntityReference(
+                id=self.service.id, type=self.service_connection.serviceType
+            ),
+        )
+
+    def create_database_service(self, dbService):
+        ATLAS_DEFAULT_CONFIG["type"] = dbService
+        ATLAS_DEFAULT_CONFIG["serviceName"] = dbService
+        config = WorkflowSource.parse_obj(ATLAS_DEFAULT_CONFIG)
+        create_service_entity = self.metadata.get_create_service_from_source(
+            entity=DatabaseService, config=config
+        )
+
+        yield create_service_entity
+        logger.info(f"Created Database Service {ATLAS_DEFAULT_CONFIG['serviceName']}")
+        self.service = self.metadata.get_by_name(
+            entity=DatabaseService, fqn=self.service_connection.dbService
+        )
+
+    def create_database_entity(self, database):
+        try:
+            self.status.scanned(database)
+            yield CreateDatabaseRequest(
+                name=database,
+                description=database,
+                service=EntityReference(
+                    id=self.service.id,
+                    type="databaseService",
+                ),
+            )
+        except Exception as e:
+            logger.debug(traceback.format_exc())
+            logger.error(f"Failed to create database entity, due to {e}")
+            self.status.failure(database["name"], str(e))
+            return None
+
+    def create_message_service(self, messagingService):
```

Comment:
Let's not create any service from here 🙏 Let's ask users to do that manually before and we just link the data there.

Also, I think you were aiming to ingest Atlas data in a MySQL service? Looks like we have some topic data as well

Reply:
@pmbrull 
We will ask user to create service, `dbService`  and `messagingService` will retrieve the service name from config for database and topic to execute ingestion.